ibm/service/atracker/resource_ibm_atracker_target.go

```diff
@@ -487,6 +553,16 @@ func resourceIBMAtrackerTargetLogdnaEndpointPrototypeToMap(model *atrackerv2.Log
 	return modelMap, nil
 }
 
+func resourceIBMAtrackerTargetEventstreamsEndpointPrototypeToMap(model *atrackerv2.EventstreamsEndpoint) (map[string]interface{}, error) {
+	modelMap := make(map[string]interface{})
+	modelMap["target_crn"] = model.TargetCRN
+	modelMap["brokers"] = model.Brokers
+	modelMap["topic"] = model.Topic
+	// TODO: remove after deprecation
+	modelMap["password"] = REDACTED_TEXT // pragma: whitelist secret
```

Comment:
I think since you are storing the password as "REDACTED_TEXT" when you run terraform plan next command It  always shows a diff

Reply:
Let say user intial provided the value for password = "hello" from his .tf file . When user runs terraform apply it provision the resource and in Read method the value for password is set as "REDACTED" in statefile. In next run it always shows a diff since the password is different in .tf and statefile. 

I think you need to add DiffSuppressFunc: flex.ApplyOnce, similar to ingestion_key under logdna_endpoint argument

Reply:
Updated it